vortex-api/src/main/java/mil/army/usace/hec/vortex/VortexDataType.java

```diff
@@ -19,7 +19,8 @@ public enum VortexDataType {
 
     public static VortexDataType fromString(String dataTypeString) {
         for (VortexDataType type : values()) {
-            if (dataTypeString.equals(type.getDssString()) || dataTypeString.equals(type.getNcString()))
+            dataTypeString = dataTypeString.toLowerCase();
```

Comment:
Make the logic more robust: case-insensitive, and `contains` instead of `equals`